tutorials/MD/03_MD_Setup_Production_Run.ipynb

```diff
@@ -0,0 +1,274 @@
+    "\n",
+    "`JOBID PARTITION     NAME     USER ST       TIME  NODES NODELIST(REASON)`\n",
+    "\n",
+    "If you made a mistake and need to cancel this job, type:\n",
+    "\n",
+    "`% scancel JOBID`\n",
+    "\n",
+    "`JOBID` should be replaced by the ID of this particular job (printed in the first column; see above).\n",
+    "\n",
+    "While the job is running, you can monitor its progress by typing: \n",
+    "\n",
+    "`% tail -n 12 em.log`\n",
+    "\n",
+    "This will show you what step the simulation is at or if it has finished.\\\n",
+    "Once the energy minimization has finished, we can examine it in terms of the potential energy versus the number of minimization steps:\n",
+    "\n",
+    "`% gmx energy -s em.tpr -f em.edr -o em_potential_energy.xvg`\n",
+    "\n",
+    "type in 10 when prompted (should correspond to potential energy from the list of options presented), then a zero to finish.\\\n",
+    "The plot should look like this one:\n",
+    "\n",
+    "<img src=\"em_potential_energy.png\" width=500 height=400/>\n",
+    "\n",
+    "### Production Run\n",
+    "\n",
+    "At this stage, we would normally run a short simulation where the protein atoms are restrained while the water molecules\n",
+    "and ions are allowed to freely move around and equilibrate around the protein. For this tutorial, we will skip this bit due to\n",
+    "limited time. Now finally let us perform some molecular dynamics:\n",
+    "\n",
+    "`% gmx grompp -c em.gro -p ../setup/topol.top -f /home/shared/data/md.mdp -maxwarn 1 -o md.tpr`\n",
+    "\n",
+    "`% cp /home/shared/data/submit_md.sh .`\n",
+    "\n",
+    "`% sbatch submit_md.sh`\n",
+    "\n",
+    "You can check the status of the job again as described previously.\n",
+    "\n",
+    "At the moment it is set up to run for 1000 ps. This will take several minutes to complete depending on the waiting time in the queue - time for lunch! You don't have to wait for it to finish completely though, although now might be a good time for a break to allow at least some data to appear. The analysis can be done on the output files that will be generated or you can always use the \"one I made earlier\" in the directory /home/shared/prerun/run (This is 1000 ps simulation of the same system).\n",
+    "\n",
+    "After the end of the production run, it would be useful to obtain some properties that will give us an insight into our protein system.\\\n",
+    "There are various so-called ensembles that are used for protein simulations - probably the most common is a system where the number of particles, the pressure and the temperature are held constant (NPT). This is usually achieved by means of a heat-bath. Nevertheless, it is usually a good idea to check these as a function of time through the trajectory just to make sure nothing unexpected happened. First let us check the temperature of our simulation.\n",
```

Comment:
The "This is usually achieved by means of a heat-bath. " just seems a little bit out of place. Can we just say something like "as detailed in the .mdp file, here we use the V-rescale thermostat (link to paper), which controls the simulation temperature through the use of a heat bath coupled with a stochastic rescaling algorithm. Whilst it is quite a robust algorithm, it is usually a good..."

Reply:
More info on v-rescale added.